charts/k8s-watcher/kube-install/k8s-watcher/templates/clusterrole.yaml

```diff
@@ -5,215 +5,223 @@ metadata:
     resources:
-    - nodes
-    - pods
+      - nodes
+      - pods
     verbs:
-    - get
-    - watch
-    - list
+      - get
+      - watch
+      - list
   - apiGroups:
-    - admissionregistration.k8s.io
+      - admissionregistration.k8s.io
     resources:
-    - mutatingwebhookconfigurations
-    - validatingwebhookconfigurations
+      - mutatingwebhookconfigurations
+      - validatingwebhookconfigurations
     verbs:
-    - get
-    - watch
-    - list
+      - get
+      - watch
+      - list
   - apiGroups:
-    - authorization.k8s.io
+      - authorization.k8s.io
     resources:
-    - localsubjectaccessreviews
-    - selfsubjectaccessreviews
-    - selfsubjectrulesreviews
-    - subjectaccessreviews
+      - localsubjectaccessreviews
+      - selfsubjectaccessreviews
+      - selfsubjectrulesreviews
+      - subjectaccessreviews
     verbs:
-    - get
-    - watch
-    - list
+      - get
+      - watch
+      - list
   - apiGroups:
-    - autoscaling
+      - autoscaling
     resources:
-    - horizontalpodautoscalers
+      - horizontalpodautoscalers
     verbs:
-    - get
-    - watch
-    - list
+      - get
+      - watch
+      - list
   - apiGroups:
-    - certificates.k8s.io
+      - certificates.k8s.io
     resources:
-    - certificatesigningrequests
+      - certificatesigningrequests
     verbs:
-    - get
-    - watch
-    - list
+      - get
+      - watch
+      - list
   - apiGroups:
-    - coordination.k8s.io
+      - coordination.k8s.io
     resources:
-    - leases
+      - leases
     verbs:
-    - get
-    - watch
-    - list
+      - get
+      - watch
+      - list
   - apiGroups:
-    - discovery.k8s.io
+      - discovery.k8s.io
     resources:
-    - endpointslices
+      - endpointslices
     verbs:
-    - get
-    - watch
-    - list
+      - get
+      - watch
+      - list
   - apiGroups:
-    - flowcontrol.apiserver.k8s.io
+      - flowcontrol.apiserver.k8s.io
     resources:
-    - flowschemas
-    - prioritylevelconfigurations
+      - flowschemas
+      - prioritylevelconfigurations
     verbs:
-    - get
-    - watch
-    - list
+      - get
+      - watch
+      - list
   - apiGroups:
-    - node.k8s.io
+      - node.k8s.io
     resources:
-    - runtimeclasses
+      - runtimeclasses
     verbs:
-    - get
-    - watch
-    - list
+      - get
+      - watch
+      - list
   - apiGroups:
-    - policy
+      - policy
     resources:
-    - poddisruptionbudgets
-    - podsecuritypolicies
+      - poddisruptionbudgets
+      - podsecuritypolicies
     verbs:
-    - get
-    - watch
-    - list
+      - get
+      - watch
+      - list
   - apiGroups:
-    - scheduling.k8s.io
+      - scheduling.k8s.io
     resources:
-    - priorityclasses
+      - priorityclasses
     verbs:
-    - get
-    - watch
-    - list
+      - get
+      - watch
+      - list
+  - apiGroups:
+      - apps
+    resources:
+      - deployments
+      - replicasets
+      - statefulsets
+    verbs:
+      - patch
```

Comment:
What about the helm chart?
You updated auto-generated files here.

You should update the helm templates, and run `make docker-generate-kube` to generate the static kubernetes manifests.
The kustomize manifests have to be updated manually for now as well (in the root of the repo)